api/internal/handler/upstream/upstream.go

```diff
@@ -17,11 +17,11 @@
 package upstream
 
 import (
+	"encoding/json"
 	"net/http"
 	"reflect"
 	"strings"
 
-	"github.com/api7/go-jsonpatch"
```

Comment:
seems we don't need `go-jsonpatch` any more. try `go mod tidy` to see if we could remove it？